snakedeploy/scaffold_plugins/common.py

```
@@ -100,6 +112,10 @@ def save_pyproject(pyproject):
             ["pixi", "task", "add", "--feature", "dev", "format", "ruff format"],
             check=True,
         )
+        sp.run(
+            ["pixi", "task", "add", "--feature", "dev", "typecheck", "pyrefly check"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)